### PR TITLE
Greeting page

### PR DESCRIPTION
**Pull request by reverbePrintemps**
# Context

This PR aims to tackle [the greeting page challenge](https://github.com/spiced-academy/honey-web-dev/blob/main/sessions/js-functions-2/challenges-js-functions-2.md#greeting-page).

# Changes

- [x] Implemented `getGreeting` function
- [x] Implemented `getDayColor` function

# How to run

1. Clone repo
2. Checkout this branch
3. Run index.js file

# TODO

- [ ] Get PR reviewed

# GIF

### CODE REVIEW

**Comment by spacedoe** [comment]
Just a few minor comments, otherwise good to merge!

**Comment by reverbePrintemps** [comment]
@spacedoe Again, I'm not seeing your comments 😭 I wonder what's going on 🤔 We should probably sit down together tomorrow and figure this out 😅

File: challenges/js-functions-2/js-functions-2_greeting-page/js/index.js [hunk 1]
```diff
@@ -1,34 +1,39 @@
...
-}
-
-function getDayColor() {
-  // Code here
-}
+const getGreeting = () => {
+  const hour = new Date().getHours();
+  let greeting = "";
+  if (hour >= 6 && hour < 12) {
+    greeting = "Good Morning";
```
[the remark this answers is not in the document]
Comment (reverbePrintemps) [reply challenges/js-functions-2/js-functions-2_greeting-page/js/index.js:7]: Good catch! : )

File: challenges/js-functions-2/js-functions-2_greeting-page/js/index.js [hunk 2]
```diff
@@ -1,34 +1,39 @@
...
+  } else if (hour >= 12 && hour < 18) {
```
[the remark this answers is not in the document]
Comment (reverbePrintemps) [reply challenges/js-functions-2/js-functions-2_greeting-page/js/index.js:8]: Indeed, thanks for spotting!

File: challenges/js-functions-2/js-functions-2_greeting-page/js/index.js [hunk 3]
```diff
@@ -1,34 +1,39 @@
...
+    greeting = "Good Afternoon";
+  } else if (hour >= 18 && hour < 22) {
```
[the remark this answers is not in the document]
Comment (reverbePrintemps) [reply challenges/js-functions-2/js-functions-2_greeting-page/js/index.js:10]: Good catch, thanks!

**Pull request merged**